src/data/value/into.rs

```diff
@@ -0,0 +1,91 @@
+use {
+    super::{Value, ValueError},
+    crate::result::Result,
+};
+
+impl Into<String> for Value {
+    fn into(self) -> String {
+        match self {
+            Value::Str(value) => value,
+            Value::Bool(value) => (if value { "TRUE" } else { "FALSE" }).to_string(),
+            Value::I64(value) => value.to_string(),
+            Value::F64(value) => value.to_string(),
+            Value::Null => String::from("NULL"),
+        }
+    }
+}
+
+pub trait TryInto<T>: Sized {
+    fn try_into(&self) -> Result<T>;
+}
```

Comment:
Let's simply use `std::convert::TryInto` rather than the custom one.

Then, you might need to change a bit but it will be ok.

![image](https://user-images.githubusercontent.com/2025065/112339212-cfc24b00-8d02-11eb-9eaf-8dab2c7c46b4.png)

Reply:
Oh oops! Thanks haha